src/go/rpk/go.mod

```diff
@@ -1,82 +1,82 @@
 module github.com/redpanda-data/redpanda/src/go/rpk
 
-go 1.22.2
+go 1.23.1
 
 // add the git commit hash as the target version and `go mod tidy` will transform it into pseudo-version
 replace github.com/hamba/avro/v2 => github.com/redpanda-data/go-avro/v2 v2.0.0-20240405204525-77b1144dc525
 
 require (
-	buf.build/gen/go/redpandadata/cloud/connectrpc/go v1.17.0-20241024195046-353ea4645e3d.1
-	buf.build/gen/go/redpandadata/cloud/protocolbuffers/go v1.35.1-20241024195046-353ea4645e3d.1
-	buf.build/gen/go/redpandadata/common/protocolbuffers/go v1.35.1-20240917150400-3f349e63f44a.1
-	buf.build/gen/go/redpandadata/dataplane/connectrpc/go v1.17.0-20241112225414-3759fedba3f3.1
-	buf.build/gen/go/redpandadata/dataplane/protocolbuffers/go v1.35.1-20241112225414-3759fedba3f3.1
-	cloud.google.com/go/compute/metadata v0.5.2
-	connectrpc.com/connect v1.17.0
+	buf.build/gen/go/redpandadata/cloud/connectrpc/go v1.18.1-20241024195046-353ea4645e3d.1
+	buf.build/gen/go/redpandadata/cloud/protocolbuffers/go v1.36.5-20241024195046-353ea4645e3d.1
+	buf.build/gen/go/redpandadata/common/protocolbuffers/go v1.36.5-20240917150400-3f349e63f44a.1
+	buf.build/gen/go/redpandadata/dataplane/connectrpc/go v1.18.1-20250224144801-a7e50018d922.1
+	buf.build/gen/go/redpandadata/dataplane/protocolbuffers/go v1.36.5-20250224144801-a7e50018d922.1
+	cloud.google.com/go/compute/metadata v0.6.0
+	connectrpc.com/connect v1.18.1
 	github.com/AlecAivazis/survey/v2 v2.3.7
 	github.com/avast/retry-go v3.0.0+incompatible
-	github.com/aws/aws-sdk-go v1.55.5
+	github.com/aws/aws-sdk-go v1.55.6
 	github.com/beevik/ntp v1.4.3
 	github.com/bufbuild/protocompile v0.14.1
 	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/docker/docker v27.2.1+incompatible
 	github.com/docker/go-connections v0.5.0
 	github.com/docker/go-units v0.5.0
-	github.com/fatih/color v1.17.0
-	github.com/hamba/avro/v2 v2.25.2
+	github.com/fatih/color v1.18.0
+	github.com/hamba/avro/v2 v2.28.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
 	github.com/kr/text v0.2.0
 	github.com/lestrrat-go/jwx v1.2.30
-	github.com/linkedin/goavro/v2 v2.13.0
+	github.com/linkedin/goavro/v2 v2.13.1
 	github.com/lorenzosaino/go-sysctl v0.3.1
 	github.com/mattn/go-isatty v0.0.20
-	github.com/moby/term v0.5.0
+	github.com/moby/term v0.5.2
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0
 	github.com/pkg/browser v0.0.0-20240102092130-5ac0b6a4141c
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_model v0.6.1
-	github.com/prometheus/common v0.60.1
+	github.com/prometheus/common v0.62.0
 	github.com/redpanda-data/common-go/rpadmin v0.1.13
 	github.com/rs/xid v1.6.0
-	github.com/safchain/ethtool v0.4.1
+	github.com/safchain/ethtool v0.5.10
```

Comment:
it seems bumping [safchain/ethtool](https://github.com/safchain/ethtool) from 0.4.1 to 0.5.10 [will break darwin](https://github.com/safchain/ethtool/commit/4d519940893ff61cf8d386fca63940c205edd44f#commitcomment-150901849)

Reply:
i opened jira to track `dev` branch changes that were merged: [UX-92]